### ISSUE TITLE
Multiple tags not parsed nor serialized properly

### ISSUE THREAD

**Issue by gracinet** [bjonnh/PyOrgMode#27]
Hi, and thank you for this useful library.
On a node with multiple tags, such as this:

```
     ** Renew servers :work:urgent:
```

What I get is an `OrgElement` with tags attribute `['work', 'urgent']` (fine) but with heading `Renew servers :work`

On the serialization side, if I create an `OrgElement` with tags `['work', 'urgent']` and heading `Renew servers`, I'll get this:

```
     ** Renew servers :work::urgent:
```

I have some [working code](https://github.com/gracinet/PyOrgMode/commit/5a0b2969069152e6bb14d4455b45f4f114a56c48) that corrects this, that I'm willing to submit properly, but I noticed while reviewing the existing tests `test.org` has such a line.

```
* TAGS TEST :TAG1::TAG2:
```

Since I'm a somewhat limited user of org-mode, I wonder if there's anything actually intentional in this

**Comment by salomonderossi** [comment]
Same issue here. The patch from @gracinet worked for me :+1:

**Comment by bjonnh** [comment]
Hi, 
Thanks for the comments and sorry for the delay.

I need to see if it works with another correction on tag handling.

Stay tuned.

**Comment by gracinet** [comment]
Hi, no worries, and as a matter of coincidence I'm starting over tracking my time with org-mode, so the timing is perfect.

**Comment by bjonnh** [comment]
This is solved by #34, please tell me if you still have issues.
